### DISCUSSION TITLE
PLEASE I NEED HELP WITH AUDIO! (Fixed! Thank you hajimehoshi! :D)

### DISCUSSION THREAD

**Discussion by eboatwright** [hajimehoshi/ebiten#1777] [Q&A]
Please, I need this for a game jam. I need the simplest possible way to play a .wav file in ebiten. :)

**Comment by lauchimoon** [comment]
Please check the examples before posting an issue like this.
https://github.com/hajimehoshi/ebiten/blob/main/examples/audio/main.go

**Comment by eboatwright** [comment]
I tried that, and it didn't work

**Reply by hajimehoshi** [reply]
https://github.com/hajimehoshi/ebiten/blob/main/examples/wav/main.go is the simplest example to play wav

**Reply by eboatwright** [reply]
ok

**Reply by eboatwright** [reply]
It wasn't fully functional but I fixed a few things and it actually works! Thank you! :smile:
